README.adoc

```diff
@@ -65,9 +65,9 @@ If you want to raise an issue, please follow the recommendations below:
 
 * Before you log a bug, please search the {github}/issues[issue tracker] to see if someone has already reported the problem.
 * If the issue doesn't already exist, {github}/issues/new[create a new issue].
-* Please provide as much information as possible with the issue report, we like to know the version of Spring Boot that you are using, as well as your Operating System and JVM version.
-* If you need to paste code, or include a stack trace use Markdown +++```+++ escapes before and after your text.
-* If possible try to create a test-case or project that replicates the problem and attach it to the issue.
+* Please provide as much information as possible with the issue report. We like to know the Spring Boot version you are using, your Operating System, and the JVM version.
```

Comment:
You can drop "you are using" and make "your Operating System" be "operating system." Note that "operating system should not be capitalized.